src/validators/RequestSchema.js

```diff
@@ -249,7 +249,7 @@ const UpdateSettingsRequestSchema = Joi.object().keys({
     ),
   }),
   favicon: Joi.string(),
-  "facebook-pixel": Joi.string().allow(""),
+  "facebook-pixel": Joi.number().unsafe().allow(null),
```

Comment:
Why arent we just using 
`Joi.string(),Joi.number()` instead ah? 
eg something like how the docs [define the access token](https://joi.dev/api/?v=17.9.1#example)

Reply:
A 15 digit number is too large to be verified using Joi.number() alone! .unsafe allows for larger numbers, see https://github.com/hapijs/joi/blob/master/API.md#numberunsafeenabled

Reply:
Hmm wouldnt it then be concerning if we are handling numbers > Number.MAX_SAFE_INTEGER? should we treat it as a string instead?

Reply:
This is true, but Kish made a good point that we might want to treat it as a string instead - given that it's an id, we should be treating it as a string instead of a number, i've reverted it to string in https://github.com/isomerpages/isomercms-backend/pull/776/commits/058b761ba760f8489dc97c9c7b32290c430b1148 and parsed int to string instead!